apps/web/src/components/(dashboard)/layout/profile-dropdown.tsx

```diff
@@ -118,7 +118,7 @@ export const ProfileDropdown = ({ user }: ProfileDropdownProps) => {
 
         <DropdownMenuItem
           onSelect={() =>
-            signOut({
+            void signOut({
               callbackUrl: '/',
             })
```

Comment:
I was curious when I made some changes to these files, Is there any specific reason we are handling the onSelect like this instead of the more traditional way of handleSubmit and creating a new function for it? or is it to prevent the infinite loop?

Reply:
For onselect specifically it’s to have commands on the item rather than normal handlers that have to route it